accounts/aws/patch-aws.yml

```diff
@@ -8,7 +8,21 @@ kind: SpinnakerService
 metadata:
   name: spinnaker
 spec:
-  spinnakerConfig:
+  spinnakerConfig:  
+    profiles:  
+      deck:                                     # Enables Lambda Functions UI
+        settings-local.js: |
+          window.spinnakerSettings.feature.functions = true
```

Comment:
@away168 @german-muzquiz Should this be moved out to it's own patch file since kustomize can't merge string changes?

Reply:
oh yeah, we had a comment about this in slack - so maybe add a new file with comment in core_config

Reply:
and make a comment in both places (so its easier to discover)

Reply:
Yes I think this would be the best. @german-muzquiz what do you think?

Reply:
I totally missed the conversation, I was OOO when it started.

Yeah I think it makes sense to have this file in its own patch and other reference it. It's a technical limitation today that when cannot merge `settings-local.js` because it's a big string. It's the same case with patches of the `kustomize` section.